styles/Editor.css

```diff
@@ -939,6 +939,47 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer {
+  font-size: 14px;
+}
+
+.ProseMirror .table-footer > div:first-child {
+  padding-top: 8px;
+}
+
+.ProseMirror .table-footer > div:not(:first-child)::before {
+  content: '';
+  border-bottom: 1px dashed #C9C9C9;
+  display: block;
+  margin-bottom: 8px;
+}
+
+.ProseMirror .table-footer > div {
+  counter-reset: table_footnote;
+}
+
+.ProseMirror .table-footer > div > div > p::before {
+  counter-increment: table_footnote;
+  content: counter(table_footnote);
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  vertical-align: super;
+  font-size: 75%;
+  margin-right: 4px;
+}
+
+.ProseMirror .table-footer > div > div > p {
+  margin-bottom: 0px;
+  margin-top: 0px;
+}
+
+.ProseMirror .table-footer > div > div.footnote-selected {
```

Comment:
It would be preferable not to use generic tags and direct descendants as any changes in html will require more similar selector. Would be nice to have a more loose selector here and on other similar places. Something close to 
.ProseMirror .table-footer .footnote-selected, if possible.

Reply:
You are completely right. I will modify it.

Reply:
Done!